src/Avalonia.Themes.Fluent/Controls/SplitButton.xaml

```diff
@@ -32,7 +32,7 @@
 
   <Style Selector="SplitButton">
     <Setter Property="Background" Value="{DynamicResource SplitButtonBackground}" />
-    <Setter Property="Foreground" Value="{DynamicResource SplitButtonForeground}" />
+    <Setter Property="TextElement.Foreground" Value="{DynamicResource SplitButtonForeground}" />
```

Comment:
Doesn't matter if PathIcon is a TextElement or not.
"TextElement" here references where this attached property is defined - in TextElement class.
Any other control has a reference-copy of this property with another owner (using "AddOwner" call). Or they don't even need to have this copy, because it is an attached property.

Since property is still the same, this Setter will continue to work.
Basically, that's why I was asking to use AddOwner where it's possible in SplitButton before, so same properties can be reused not only in code behind, but also in styles.

Reply:
I believe this PR makes this property to match WPF definition:
https://github.com/dotnet/wpf/blob/main/src/Microsoft.DotNet.Wpf/src/PresentationFramework/System/Windows/Documents/TextElement.cs#L723 (also in TextElement class).

Reply:
I actually not sure why we use full path to the property "TextElement.Foreground" instead of shorter "Foreground". Since we already have this prop on TemplatedControl via AddOwner.

Fine for me, but I wonder if there is a specific reason. @Gillibald

Reply:
XamlX comlained otherwise

Reply:
This
```
<Style Selector="SplitButton /template/ Button#PART_PrimaryButton:pointerover /template/ ContentPresenter,
         SplitButton /template/ Button#PART_SecondaryButton:pointerover /template/ ContentPresenter">
		 <Setter Property="Foreground" Value="{DynamicResource SplitButtonForegroundPointerOver}" />
```
isn't compiling because ContentPresenter is just a control and not a TemplatedControl.

I need to use `TextElement.Foreground` instead.

Reply:
Oh, I remember now. ContentPresenter doesn't have foreground at all, yes.
So, ```TextElement.Foreground``` is needed here as an inherited value.